app/src/main/java/com/example/habitsmasher/ui/history/AddHabitEventDialog.java

```diff
@@ -56,10 +45,8 @@
 
     /**
      * Default constructor
-     * @param username (String) The username of the user who is adding events
      */
-    public AddHabitEventDialog(String username) {
-        _username = username;
+    public AddHabitEventDialog() {
```

Comment:
Do we still need this constructor if it's empty? We usually need empty constructors for firebase, but this doesn't look like a class that's modelled in the db.